server/src/test/java/org/eclipse/lsp/cobol/usecases/engine/UseCase.java

```diff
@@ -48,7 +48,8 @@
   @Builder.Default List<EmbeddedCodeNode.Language> features = Collections.emptyList();
   /** Analysis dialects */
   @Builder.Default List<String> dialects = Collections.emptyList();
-
+  /** User predefined labels */
+  @Builder.Default List<String> predefinedLabels = Collections.emptyList();
```

Comment:
It's really great that you add it here for UseCase.
But you didn't use it. Can you, please, add a UseCase test for predefined labels?

Reply:
Done